### ISSUE TITLE
💡 Add the ability to always have a message at the bottom in an auto-thread channel

### ISSUE THREAD

**Issue by MarcusOtter** [MarcusOtter/discord-needle#256]
### Describe the improvement

I've been wanting to support something like this for a while. Many people don't know how auto-thread channels work. The TypeScript server has a decent solution to this, I think (albeit the message itself isn't super helpful):

![image](https://user-images.githubusercontent.com/35617441/184697231-845e0254-39ce-4d93-85da-5a5600f36934.png)

This message always stays at the bottom of the channel, so when new messages are posted and threads are created, the bot deletes the old message and moves it to the bottom. This option should be off by default and the message should obviously be configurable.

### Problems this improvement solves

Helps people understand what will happen in auto-thread channels, and lets servers customize their auto-thread experience even more. It will also be helpful e.g. for instructing users to put their thread title on the first line of the message, if the server has that configured (WIP on #250).

### Alternative solutions

N/A

**Comment by MarcusOtter** [comment]
Another (better) example is the Gitpod Discord that does this as well, with a more helpful message. I don't know what actually happens if I write here (maybe it auto-threads?) but the "Ask a question" button opens a modal, like the #205 feature request.

![image](https://user-images.githubusercontent.com/35617441/184735848-71db17c5-2653-45e4-8bdc-b270a1b8ee3b.png)

This issue is just for the support of having a message at the bottom with some content though.

**Comment by Madis0** [comment]
That would be great indeed, currently using https://www.stickybot.info/ for that.

**Comment by MarcusOtter** [comment]
Usually, if there are other simple bots that accomplish exactly the feature I want, I would close the issue and tell people to use that. But wow, I really can't recommend that bot to anyone 😅. Thanks for linking it.

**Comment by ririko5834** [comment]
Yeah, I like this to have button to open thread sticky message in thread channel. There could be Discord modal - form where user can enter thread name (title) and thread description (first message) and bot should make the thread and send the first message that user has filled in the form to the thread channel
